plugins/lookup/consul_kv.py

```diff
@@ -162,30 +165,6 @@ def run(self, terms, variables=None, **kwargs):
                         values.append(to_text(results[1]['Value']))
         except Exception as e:
             raise AnsibleError(
-                "Error locating '%s' in kv store. Error was %s" % (term, e))
+                "Error locating '%s' in kv store. Error was %s" % (key, e))
 
         return values
-
-    def parse_params(self, term):
-        params = term.split(' ')
```

Comment:
By removing this and ignoring the optional parameters per term, you are breaking potential users of this lookup. Please undo this removal.

Reply:
see ansible/ansible#74108, you can 'move' to set_options/get_option and keep some backwards compatibility to the existing parsing, but I recommend deprecating it and move to the normalized interface for all lookups.

Reply:
Yes, but deprecation should only happen some time after adding the new parsing (say one major version later).